scipy/stats/boost/tests/test_boost_stats.py

```diff
@@ -0,0 +1,77 @@
+'''Boost stats tests.'''
```

Comment:
When I try to run this with `runtests.py`, I get 
```
ERROR: module or package not found: scipy/stats/boost/tests/test_boost_stats.py (missing __init__.py?)
```

Reply:
Probably same issue as before, just added the boost/tests directory as another submodule